test/parallel/test-http-server-unconsume-consume.js

```diff
@@ -4,8 +4,8 @@ const http = require('http');
 
 const testServer = http.createServer(common.mustNotCall());
 testServer.on('connect', common.mustCall((req, socket, head) => {
-  socket.write('HTTP/1.1 200 Connection Established' + '\r\n' +
-      'Proxy-agent: Node-Proxy' + '\r\n' +
+  socket.write('HTTP/1.1 200 Connection Established\r\n' +
+      'Proxy-agent: Node-Proxy\r\n' +
```

Comment:
No, [it spares strings in different lines:](http://eslint.org/docs/rules/no-useless-concat)

> Examples of correct code for this rule... when the string concatenation is multiline